pkgs/development/python-modules/gistyc/default.nix

```diff
@@ -21,6 +21,10 @@ buildPythonPackage rec {
     requests
   ];
 
+  pythonImportsCheck = [
+    "gistyc"
+  ];
```

Comment:
The repository seems to have some tests. Can't we use them instead :thinking: ?

https://github.com/ThomasAlbin/gistyc/tree/main/tests

Reply:
Looks like they need networking.